src/shared/containers/SetOrg.tsx

```diff
@@ -296,12 +295,7 @@ const SetOrg: FC = () => {
           />
 
           {/* Getting Started */}
-          {isFlagEnabled('firstMile') ? (
-            <Route exact path="/orgs/:orgID" component={HomepageContainer} />
-          ) : (
-            <Route exact path="/orgs/:orgID" component={MePage} />
-          )}
-
+          <Route exact path="/orgs/:orgID" component={HomepageContainer} />
           {isFlagEnabled('firstMile') && [
```

Comment:
If `HomepageContainer` is what will be shown on the home page (despite firstMile ff being on or off), should we also remove this ff here?

Reply:
oh wow, nice find. I totally missed this.

Reply:
noooice!